capture/db.c

```diff
@@ -1323,6 +1323,28 @@ void arkime_db_save_session(ArkimeSession_t *session, int final)
 
             BSB_EXPORT_rewind(jbsb, 1); // Remove last comma
             BSB_EXPORT_cstr(jbsb, "],");
+
+            break;
+        }
+        case ARKIME_FIELD_TYPE_OBJECT: {
+            ArkimeFieldObjectHashStd_t *ohash = session->fields[pos]->ohash;
+            ArkimeFieldObjectSaveFunc saveCB = config.fields[pos]->object_save;
+            ArkimeFieldObjectFreeFunc freeCB = config.fields[pos]->object_free;
+
+            BSB_EXPORT_sprintf(jbsb, "\"%sCnt\":%d,", config.fields[pos]->dbField, HASH_COUNT(o_, *ohash));
+            BSB_EXPORT_sprintf(jbsb, "\"%s\":[", config.fields[pos]->dbField);
+
+            ArkimeFieldObject_t *object;
+
+            HASH_FORALL_POP_HEAD2(o_, *ohash, object) {
+                saveCB(&jbsb, object, session);
+                freeCB(object);
```

Comment:
I know certs doesn't do it, but lets add if(freeField) here and the ohash free below

Reply:
Sure, sounds like a good idea, hadn't really noticed it in the other types.